minigotchi-ESP32/display.cpp

```diff
@@ -3,58 +3,97 @@
 */
 
 #include "display.h"
+#include <TFT_eSPI.h>  // Include the TFT_eSPI library to enable control of TFT displays using the ESP32 and CYD
```

Comment:
i like to include relevant screen libraries in the header, it would help maintain the organization and code structure

Reply:
can you remove that since it's already in `display.h`?

Reply:
Removed.